src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1271,6 +1271,17 @@ export class AmpStoryPlayer {
     });
   }
 
+  /**
+   * @param {string} pageId
+   * @public
+   */
+  goToPageId(pageId) {
```

Comment:
We might want to join this together with the `show(story, pageId)` API, this way we can control the code flow whenever a publisher wants to switch a story AND go to a specific page. Otherwise it would be an easy way to get a race condition.